.github/workflows/reusable-unit.yml

```diff
@@ -107,8 +107,13 @@ jobs:
         run: yarn ${{ matrix.package }} build
 
       - name: Check ${{ matrix.package }} bundle size
-        if: ${{ matrix.package == 'react' }}
-        run: yarn react size
+        if: |
+          matrix.package == 'react' ||
+          matrix.package == 'react-notifications' ||
+          matrix.package == 'react-geo' ||
+          matrix.package == 'react-storage' ||
+          matrix.package == 'react-liveness'
```

Comment:
Why not include the `react-core*` packages?

Reply:
We're only adding size checks on packages we publish externally.
Changes in `react-core*` packages should be picked up and reflected on the externally published packages.

Reply:
Maybe we should consider including some size checks on the connected component utils that are exposed via `react-core`? It may help in debugging where a size increase came from, thoughts?

Reply:
Added size checks on `react-core*` packages: https://github.com/aws-amplify/amplify-ui/pull/4085/commits/9b259a19e58eebb8513bab35dce091284c682d19